src/cmake/externalpackages.cmake

```
@@ -211,8 +211,11 @@ if (OSL_USE_OPTIX)
         endif ()
 
         checked_find_package (CUDA REQUIRED
-                            VERSION_MIN 8.0
-                            PRINT CUDA_INCLUDES)
+                             VERSION_MIN 9.0
+                             RECOMMENDED_MIN 11.0
+                             RECOMMENDED_MIN_REASON
+                                "We don't actively test Cuda older than 11"
```

```suggestion
                             RECOMMEND_MIN 11.0
                             RECOMMEND_MIN_REASON
                                "We don't actively test CUDA older than 11"
```

Wrong argument name.

---

D'oh! you're right. Fixed.